docs/physics/montecarlo/propagation.rst

```diff
@@ -150,103 +168,101 @@ Thomson scattering according to
 
 .. math::
 
-    \Delta \tau = \sigma_{\mathrm{T}} l.
+    \Delta \tau = \chi_{\mathrm{T}} l.
+    
+This corresponds to the fact that a packet has a probability of :math:`1-e^{\sigma_{\mathrm{T}} n_e l}` of going through a Thomson scattering prior to traveling a distance :math:`l` (in other words, the probability of the packet making it across a distance :math:`l` without scattering is :math:`e^{\sigma_{\mathrm{T}} n_e l}`).
```

Comment:
Shouldn't it be \chi in the inline equation rather than \sigma?

Reply:
It is-- it was \sigma, which was incorrect (https://github.com/tardis-sn/tardis/blob/master/tardis/montecarlo/montecarlo_numba/r_packet.py#L220), and I changed it to \chi.

Reply:
I mean the equation in the text still shows \sigma

Reply:
Ah. That one is multiplied by n_e. I see where that can be confusing-- looking at how it's done in the code, would it be bad to just leave out the opacity and just change the formula to be \Delta\tau = n_e \sigma_T l?

Reply:
Yes, especially because that's how it is done in the code